Libs/MRML/Core/vtkMRMLClipModelsNode.cxx

```diff
@@ -54,6 +55,7 @@ void vtkMRMLClipModelsNode::WriteXML(ostream& of, int nIndent)
   of << indent << " redSliceClipState=\"" << this->RedSliceClipState << "\"";
   of << indent << " yellowSliceClipState=\"" << this->YellowSliceClipState << "\"";
   of << indent << " greenSliceClipState=\"" << this->GreenSliceClipState << "\"";
+  of << indent << " clippingMethod=\"" << this->ClippingMethod << "\"";
```

Comment:
Instead of printing a number, I believe we usually print the enum associated string value.